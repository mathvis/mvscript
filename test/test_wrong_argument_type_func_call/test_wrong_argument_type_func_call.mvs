func a(b: int) bool {
	return false;
};
a(a(5));
