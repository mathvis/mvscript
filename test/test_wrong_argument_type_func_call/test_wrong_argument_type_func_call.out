error: Expected int but got bool
 --> test/test_wrong_argument_type_func_call/test_wrong_argument_type_func_call.mvs:4:1
  |
4 | a(a(5));
  | ^
  |
hint: Try changing the argument type.
